Expecto/Expecto.fs

```diff
@@ -39,7 +55,11 @@ type FocusState =
   /// All other test marked with Normal or Pending will be ignored
   | Focused
 
-  with static member isFocused = function | Focused -> true | _ -> false
+
+type SequenceMethod =
+  | Synchronous
+  | SynchronousGroup of string
+  | InParallel
```

Comment:
What does this mean? Aren't parallel and sequenced two contradictory properties?